tailcfg/tailcfg.go

```diff
@@ -131,7 +131,8 @@ type CapabilityVersion int
 //   - 88: 2024-03-05: Client understands NodeAttrSuggestExitNode
 //   - 89: 2024-03-23: Client no longer respects deleted PeerChange.Capabilities (use CapMap)
 //   - 90: 2024-04-03: Client understands PeerCapabilityTaildrive.
-const CurrentCapabilityVersion CapabilityVersion = 90
+//   - 91: 2024-04-18: Client understands NodeAttrLogExitFlows
+const CurrentCapabilityVersion CapabilityVersion = 91
```

Comment:
I think we should bump the version in the PR that implements _using_ the attr, rather than just having the attr defined. Otherwise there will be some version 91 unstable clients that ignore the attr when it's set.

Reply:
got it agreed will do